.claude/commands/bdd.md

```
@@ -0,0 +1,97 @@
+| 項目               | 内容        |
+| ------------------ | ----------- |
+| **ユースケース数** | **{N}件**   |
+| **シナリオ数**     | **{M}件**   |
+
+## 1. ユースケース一覧
+...
+
+## 2. ユースケース詳細
+
+### UC-01: {ユースケース名}
+...
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)